src/OpenSSL/SSL.py

```diff
@@ -2343,7 +2388,7 @@ def DTLSv1_get_timeout(self) -> Optional[int]:
         else:
             return None
 
-    def DTLSv1_handle_timeout(self) -> bool:
+    def DTLSv1_handle_timeout(self) -> bool:  # type: ignore[return]
```

Comment:
We ignore the return type, because `mypy` is unable to know that `_raise_ssl_error` is sure to bomb.

Reply:
Wouldn't that cause a drop in code coverage?

Reply:
That's fixed with `  # pragma: no cover` or by adding
```
[tool.coverage.report]
exclude_also = [
    "assert False",
    ]
```
to pyproject.toml. This `assert False` pattern is what's recommended by  https://typing.readthedocs.io/en/latest/source/unreachable.html#marking-code-as-unreachable.

Reply:
Isn't this what typing.NoReturn is for?

Reply:
> Isn't this what typing.NoReturn is for?

Not quite, unfortunately our `self._raise_ssl_error` does not *unconditionally* raise. It's perfectly legal to call it with `SSL_ERROR_NONE` in which case it doesn't raise.